operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/workflow/DependentResourceNode.java

```diff
@@ -54,6 +54,9 @@ public Optional<Condition<R, P>> getReconcilePrecondition() {
     return Optional.ofNullable(reconcilePrecondition);
   }
 
+  public String getName() {
```

Comment:
This is not very good practice in my opinion, it kinda implies that it's the name of the dependent node not the dependent resource. I would either remove this, and always access the name though DependentResource or rename this to `getDependentResourceName()`

Reply:
`DependentResourceNode` is an internal class so I'd argue that it doesn't really matter and since Dependent Resources always have names now, the name of the node is the name of the associated dependent, which this method captures. This certainly could be reverted but, to me, it's pretty clear that the name of the DRN is the name of the associated DR. :)

Reply:
pls do, when started looking it was kinda confusing now, what now we are talking about

Reply:
This is really small thing, at least for me it is  much easier to jump to a conclusion with that pattern